.vscode/settings.json

```diff
@@ -59,5 +59,6 @@
     "*.ts": "${capture}.spec.ts,${capture}.mock.ts"
   },
   "svelte.enable-ts-plugin": true,
-  "typescript.preferences.importModuleSpecifier": "non-relative"
+  "typescript.preferences.importModuleSpecifier": "non-relative",
+  "html.customData": [".web/node_modules/vidstack/vscode.html-data.json"]
```

Comment:
Wth does this do?

Reply:
Oh it's a vite plugin and you need this to hint it at the correct type or something?

Reply:
Vidstack is a web component, and i think this adds some of the attributes/tags that vidstack ships to the vscode HTML LSP: https://github.com/microsoft/vscode-html-languageservice/blob/main/docs/customData.md